about/policy/metadata/index.md

```diff
@@ -6,109 +6,57 @@ tags:
-
-##### Summarize your Data
-{: .text-left}
-
-- What is this dataset?
-- What is the purpose of this dataset?
-
-##### Describe your Data
-{: .text-left}
-
-- What does this dataset represent?
-  - ex: road centerlines represent the centerline of a physical road
-- How was this dataset created?
-  - ex: digitized, GPS, etc.
-- How reliable and accurate is the dataset?
-  - ex: survey-grade, mapping-grade, etc.
-- What is the general update schedule for this dataset?
-  - ex: monthly, yearly, as needed, etc.
-- Are there any warnings or limitations on the data?
-  - ex: don’t use parcels as a legal boundary depiction
-  - ex: don’t use parcels as a complete dataset for geocoding
-  - ex: don’t use zip codes for demographic generalization
-- Are there any common misconceptions or assumptions about this data that should be further explained?
-  - ex: highway milepost markers are not an accurate length measurement. If a highway is lengthened or shortened, the markers are not adjusted.
-
-##### Establish Contact Info
-{: .text-left}
-
-- Who created this dataset?
-- What is the best point of contact for this dataset?
-  - if someone had further questions or would like to get involved, who should they contact?
-
-Your answers to these questions can be plugged into the basic metadata elements (discussed below). It's that simple - answer a few questions, transfer your responses, and voilà, you officially have informative, standardized metadata.
-
-_Note: If you contribute data to Utah's State Geographic Information Database (SGID), we encourage you to use our [metadata form](https://docs.google.com/forms/d/e/1FAIpQLSeJSV935GTN3djtErkBdeXkumuZvd10z2805It1-eFn0Mlo9w/viewform?usp=sf_link). It's a great way to update your metadata and make it more discoverable. The form covers all the questions above -and best of all- after you submit your responses, your portion of the process is done! AGRC, in collaboration with the DTS editor, will handle the remaining details._
-
-The above questions are designed to work with the ESRI [Item Description](https://desktop.arcgis.com/en/arcmap/latest/manage-data/metadata/metadata-standards-and-styles.htm#ESRI_SECTION1_29222DC149544E89A4147541A4ACAF86) metadata style. We have found this style to be well-balanced for creating useful, as well as achievable, metadata.
-
-Let's take a closer look at the basic elements of SGID metadata. We like to think of these as our minimum requirements.
-
-### Basic SGID Metadata
-{: .text-left}
+Effective metadata helps people understand the source of a dataset, how up-to-date it is, and any of its inherent restrictions or limitations. Listed below are the minimum metadata items that should be present in any SGID datasets. These correspond to the Esri [Item Description](https://desktop.arcgis.com/en/arcmap/latest/manage-data/metadata/metadata-standards-and-styles.htm#ESRI_SECTION1_29222DC149544E89A4147541A4ACAF86) metadata style and the metadata displayed on an item's ArcGIS Online page.
```

Comment:
```suggestion
Effective metadata helps people understand the source of a dataset, how up-to-date it is, and any of its inherent restrictions or limitations. Listed below are the minimum metadata items that should be present in any SGID datasets. These correspond to the Esri [Item Description](https://pro.arcgis.com/en/pro-app/latest/help/metadata/view-and-edit-metadata.htm) metadata style and the metadata displayed on an item's ArcGIS Online page.
```

Would this be a more up-to-date link? ArcMap is going away despite what @rkelson thinks. ;)